### PR TITLE
Added support for lovercase countries

### PR DESCRIPTION
**Pull request by dinodsaurus**

### CODE REVIEW

File: src/common/services/context/ParseAcceptLanguage.js [hunk 1]
```diff
@@ -2,12 +2,12 @@
 
 export default (acceptLanguage: ?string) => {
   let [language, territory] = ['en', 'US'];
-  // Valid acceptLanguage en_US || en-US || en-US,en;q=0.9,sl;q=0.8
-  const localeRegex = /([a-z][a-z])[-_]([A-Z][A-Z])/;
+  // Valid acceptLanguage en_us || en_US || en-US || en-US,en;q=0.9,sl;q=0.8
+  const localeRegex = /([a-z]{2})[-_]([A-Za-z]{2})/;
```
Comment (michalsanger) [inline src/common/services/context/ParseAcceptLanguage.js]: What about just `de` as an input?

Comment (dinodsaurus) [reply src/common/services/context/ParseAcceptLanguage.js]: Oh, now I see that is also supported by the standard - https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Language will implement that today 😉

**Comment by dinodsaurus** [comment]
I added `US` as a default country when you don't specify it.
Also `cz+CZ` became valid as it checks first two chars, I would say this is a feature, not a bug 😄 
What do you think @michalsanger

**Comment by michalsanger** [comment]
@dinodsaurus  `cz+CZ` why not, but could you add it into test so it's clear it's intentional?

File: src/common/services/context/__tests__/ParseAcceptLanguage.test.js [hunk 2]
```diff
@@ -9,6 +9,11 @@ describe('Parse Accept-Language', () => {
     expect(con).toBe('US');
   });
   it('should return default values for invalid lang format', () => {
+    const [lang, con] = parseAcceptLanguage('czech');
+    expect(lang).toBe('en');
+    expect(con).toBe('US');
+  });
+  it('should return default values for invalid lang format #2', () => {
     const [lang, con] = parseAcceptLanguage('cz+CZ');
```
Comment (michalsanger) [inline src/common/services/context/__tests__/ParseAcceptLanguage.test.js]: > Also cz+CZ became valid as it checks first two chars, I would say this is a feature, not a bug

?

Comment (dinodsaurus) [reply src/common/services/context/__tests__/ParseAcceptLanguage.test.js]: yep, I realized when I did this "feature" I broke the code, so I basically removed it 🤥

**Pull request merged**